### DISCUSSION TITLE
Typescript expecting more arguments

### DISCUSSION THREAD

**Discussion by dandanthedev** [pocketbase/pocketbase#2612] [Q&A] [answered]
Hi, im trying to make a pocketbase app but while trying to authenticate typescript is expecting the other (optional) elements. How can i fix this?
![image](https://github.com/pocketbase/pocketbase/assets/20974756/acc5afef-05c6-4e6f-bc55-7ac824b985a3)

**Comment by ganigeorgiev** [comment]
What version of the JS SDK do you use?

The "all-in-one" OAuth2 call was aded in v0.14+ of the JS SDK.

**Comment by dandanthedev** [comment]
Im using 0.15.0 on Svelte.

**Reply by ganigeorgiev** [reply]
It is possible to be caused by types cache of the editor, I'm not really sure.
I'm not able to reproduce it with Sublime and the TypeScript LSP plugin.

If you can reproduce it consistently, please provide more details about your setup and ideally a minimal reproducible repo and I'll try to investigate it further.

**Reply by dandanthedev** [reply, marked as answer]
A restart of my VSCode cleared the cache and fixed my problem 😄
